.golangci.yml

```diff
@@ -29,7 +29,9 @@ linters:
     - govet
     - ineffassign
     - interfacer
-    - lll
+# lll fails for some tests (thirdparty/cmdconfig/cmdsource/_test.go)
+# Figure out how to exclude files with issues.
+#    - lll
```

Comment:
I think we should disable this rule in the specific places where we have violations. The syntax is:
```
//nolint:lll
```

Reply:
Good tip: fixed it. Thanks